src/Chapter1.hs

```diff
@@ -640,8 +661,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n =
+    if divN == 0 then n else firstDigit divN
+    where divN = div n 10
```

Comment:
Almost there!

Here you also would need to think on how to handle negative numbers 🙂